Dockerfile.dev

```diff
@@ -10,11 +10,16 @@ WORKDIR /home/remix
 
 COPY ./ ./
 
-WORKDIR /home/remix/remix
-# npm ci would probably be better
-RUN npm install
+RUN npm ci
 RUN npm run build
```

Comment:
do you need to `build` here?

Reply:
With the changes to Dockerfile (which is just a production variant) this can be used as a development environment (so you build then serve that content). Naming can be changed. Dockerfile -> Dockerfile.dev and Dockerfile.dev to Dockerfile if that's clearer :)